cloudplatform/resilience/src/main/java/com/sap/cloud/sdk/cloudplatform/resilience/ResilienceDecorator.java

```diff
@@ -26,6 +27,12 @@
 @Slf4j
 public final class ResilienceDecorator
 {
+    static String LEGACY_DECORATION_STRATEGY;
```

Comment:
[minor]

Let's make it more obvious as to why this is explicitly not `final` (as anyone should expect from the name).

```suggestion
    // non final for testing
    static String LEGACY_DECORATION_STRATEGY;
```

Reply:
I just wouldn't make it a constant then.